src/Acuminator/Acuminator.Utilities/Roslyn/Semantic/PXGraph/GraphInitializerInfo.cs

```diff
@@ -0,0 +1,23 @@
+﻿using Acuminator.Utilities.Common;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
+
+namespace Acuminator.Utilities.Roslyn.Semantic.PXGraph
+{
+    public class GraphInitializerInfo
```

Comment:
It could be a struct. Well, this is debatable. For example, we widely use `foreach` instead of `for` despite the fact that `for` is preferable for performance reasons. I'll change this to struct but I think we need a code style about it.

Reply:
Yes, it's indeed debatable. Still it is very light DTO with only three fields - a very common case to use a struct.